sklearn/tests/test_multiclass.py

```diff
@@ -456,6 +458,8 @@ def test_ovr_coef_():
                          sp.issparse(ovr.coef_))
 
 
+# TODO: Remove in 0.26 when the coef_ attribute is deprecated
```

Comment:
Deprecated -> removed

Also we need to be clear about *what* exactly needs to be removed. Otherwise one may thing the whole test should be removed.

Please place this as a comment on the same line, of if it doesn't fit, say
> TODO: remove ignore_warning in 0.26 when ...

(same in other places ;) )

Reply:
I think we should remove the whole test because the use the `coef_` attribute, is it not?

Reply:
in this case yes (I haven't checked the rest in details)

Reply:
Thank you @NicolasHug for the review!

The same apply for the rest of tests.